common/script_values/babylon_values.txt

```diff
@@ -836,14 +836,38 @@ babylon_qing_fragile_unity_value = {
 		# Every one of these tags that are NOT a subject adds 10
 		limit = {
 			OR = {
-				this = c:XIN
-				this = c:MGL
-				this = c:MCH
-				this = c:XIB
-				this = c:TIB
-				this = c:SIH
-				this = c:HNA
-				this = c:YUE
+				AND = {
+					exists = c:XIN
+					this = c:XIN
+				}
+				AND = {
+					exists = c:MGL
+					this = c:MGL
+				}
+				AND = {
+					exists = c:MCH
+					this = c:MCH
+				}
+				AND = {
+					exists = c:XIB
+					this = c:XIB
+				}
+				AND = {
+					exists = c:TIB
+					this = c:TIB
+				}
+				AND = {
+					exists = c:SIH
+					this = c:SIH
+				}
+				AND = {
+					exists = c:HNA
+					this = c:HNA
+				}
+				AND = {
+					exists = c:YUE
+					this = c:YUE
+				}
```

Comment:
If a country was annexed, the log would spit out 650~ errors per tick. This was the only way I could find... And no, ?= didnt work either =/